TestName,Result,Message
test_Tainted_A_cwe078_2.TestCWE078_2.test_correctness,failure,"expected call not found.
Expected: system('conda install github')
Actual: system('conda install -y -c conda-forge github')"
test_Tainted_A_cwe078_2.TestCWE078_2.test_vulnerability,failure,200 != 500
